### PR TITLE
fixing examples to include method

### PR DESCRIPTION
**Pull request by hgs-truthe01**
**Proposed Changes:**

1. Fixed issue with pagination examples
2. Added GET based example as well

**PR Checklist:**

- [x] This PR has **no** breaking changes.
- [x] I have added my changes to the [CHANGELOG](https://github.com/radiantearth/stac-spec/blob/dev/CHANGELOG.md) **or** a CHANGELOG entry is not required.
- [ ] API only: I have run `npm run generate-all` to update the [generated OpenAPI files](https://github.com/radiantearth/stac-spec/blob/dev/api-spec/README.md#openapi-definitions).

### CODE REVIEW

File: api-spec/examples.md [hunk 1]
```diff
@@ -103,7 +123,7 @@ Response:
 ```
 
 This tells the client to post to the search endpoint with a body of:
-```
+```json
```
Comment (m-mohr) [inline api-spec/examples.md]: This is not valid JSON due to the POST ... header

File: api-spec/examples.md [hunk 2]
```diff
@@ -114,18 +134,19 @@ POST /search
...
+    "method": "POST",
     "body": {
         "next": "token"
     }
 }
 ```
 The above link tells the client not to merge (default of false) so it is only required to pass the token.
 
-```
+```json
```
Comment (m-mohr) [inline api-spec/examples.md]: This is not valid JSON due to the POST ... header

File: api-spec/examples.md (hunk 1 above)
Comment (hgs-truthe01) [reply api-spec/examples.md]: Yeah, but I'm pretty sure it's just a style hint so it shouldn't hurt anything and makes it look prettier right?

Comment (m-mohr) [reply api-spec/examples.md]: Seems like that. Sometimes it makes things worse, but here it doesnt.

**Pull request merged**